pkg/tfgen/generate_csharp.go

```diff
@@ -1061,7 +1066,7 @@ func (rg *csharpResourceGenerator) generateInputTypes(nts []*csharpNestedType) {
 	// Write each input type.
 	for _, nt := range nts {
 		contract.Assert(nt.isInput)
-		rg.generateInputType(nt.typ, true /*nested*/)
+		rg.generateInputType(nt.typ, "ResourceArgs", true /*nested*/, true /*wrapInput*/)
```

Comment:
Should this always be `ResourceArgs`? We do generate nested types for invokes as well.

Reply:
Thanks, I found an example of this. I'll change it.

Reply:
Changed now